Redefine default library directory for aarch64.

Packages use 'aarch64-poky-linux-gcc -print-multi-os-directory' to get the
value of install directory. The value is set by MULTILIB_OSDIRNAMES and default
is '/lib64'. Add the dynamic linker ld-linux-aarch64.so.1 from package eglibc
is installed to /lib64.

But the default dynamic linker defined in gcc/config/aarch64/aarch64-linux.h
is:

#define GLIBC_DYNAMIC_LINKER "/lib/ld-linux-aarch64%{mbig-endian:_be}.so.1"

Then cause all the ELF files fail to launch because fail to find the linker.

Upstream-Status: Pending

Signed-off-by: Kai Kang <kai.kang@windriver.com>
---
--- gcc-4.9.1/gcc/config/aarch64/t-aarch64-linux.orig	2014-09-19 14:38:47.550669143 +0800
+++ gcc-4.9.1/gcc/config/aarch64/t-aarch64-linux	2014-09-19 14:39:01.234669516 +0800
@@ -22,7 +22,7 @@
 LIB1ASMFUNCS = _aarch64_sync_cache_range
 
 AARCH_BE = $(if $(findstring TARGET_BIG_ENDIAN_DEFAULT=1, $(tm_defines)),_be)
-MULTILIB_OSDIRNAMES = .=../lib64$(call if_multiarch,:aarch64$(AARCH_BE)-linux-gnu)
+MULTILIB_OSDIRNAMES = .=../lib$(call if_multiarch,:aarch64$(AARCH_BE)-linux-gnu)
 MULTIARCH_DIRNAME = $(call if_multiarch,aarch64$(AARCH_BE)-linux-gnu)
 
 # Disable the multilib for linux-gnu targets for the time being; focus
